data/wanderer/wanderers start.txt

```diff
@@ -406,26 +349,47 @@ mission "Wanderers Conversation"
 			`It is possible that the worlds they currently inhabit were formerly part of Hai territory. That might explain why the Unfettered believe they have a right to take that territory back.`
 		log "People" "Iktat Rek" `Rek is a Wanderer ambassador, a new and unusual profession among a species that until recently has had no contact with aliens except for the occasional Unfettered Hai raid. His job is to help the Wanderers develop a deeper relationship with humans.`
 		conversation
-			`With the help of your new translation device, you are able to contact the Wanderer government. They welcome you back and tell you that a Wanderer named "Iktat Rek" has been appointed as the official envoy to deal with human visitors, and he should be arriving at your landing pad shortly.`
-			`	Rek is slightly larger than most of the Wanderers you have seen, with muted white and brown plumage and a calm, dignified bearing. "It is very [rare, exceptional] in our history that we have to [deal with, cope with] other species," he says, "and now we find ourselves dealing with two at once." He is speaking the Hai language, and the translation device does its best to render his words in your own language.`
+			`After you land, a group of Wanderers take you to the engineer from your last meeting. The engineer seemed rather happy as it presented the device to you, saying: "It will [function, succeed] at speeches of typical [speed, tempo], however..." They start speaking faster, demonstrating the gibberish results it produces. "This is Iktat Rek," they slow down, gesturing to another Wanderer. "He is [appointed, chosen] as the official [envoy, bearer] for human [visitors, couriers]."`
+			`	Rek is slightly larger than most of the Wanderers you have seen, with muted white and brown plumage and a calm, dignified bearing. "It is very [rare, exceptional] in our history that we have to [deal with, cope with] other species," he says, "and now we find ourselves [dealing, coping] with two at once."`
 			choice
 				`	"How did you learn the Hai language?"`
 					goto language
 				`	"How long have you been fighting the Unfettered Hai?"`
-			
+					goto fighting
+				`	"Why were you able to create a translation device so quickly?"`
+
+			# Wanderer AI technology is far beyond ours, and we can already automatically translate languages this well. Of course they can too.
+			`	"[Automated, machine] language translation is a [primitive, early] application of machine [learning, intelligence]. For us it is [elementary, instinctual] mathematics. Our [thinkers, tinkerers] told me that most of the [work, enjoyment] was adapting the [interface, conceptualization, mechanism] to a human mind."`
```

Comment:
Wait, sorry, _how_ does this translator work again? Are you hearing things from induced brainwave patterns or something?

Reply:
This is a tier 2 race that specializes in manipulating lifeforms on a planetary scale. They're not limited to primitive technologies like the humans and Hai. Certainly, a brain interface is within their abilities.

However, that was not what I was trying to imply. I'll remove the word "mind."

Reply:
Hmmm... or maybe I should double down on the brain interface? It sounds a lot more interesting and certainly within the Wanderers' abilities.

Reply:
Brain interface is almost certainly within reach of the Wanderer capacity, but it would have been clearly noticed if utilized, as well as making any other form of interface superfluous.

I don't think it's the right direction for a change here.

Let's save that for another group to explore more thoroughly.

Reply:
Considering that humanity in 2023 is already poking at the fringes of Mind-Machine-Interfaces...

As alternatives, there's the classic heads-up visual display (google-glasses style thing), or an earpiece. I'm sure the Wanderers could make either one of those that is totally invisible(too small to see, invisibly thin contact lens, etc); but the action of taking an earpiece off the device and putting it on their head could be a useful visual thing just so the reader has some means of knowing what's happening.

Reply:
The word "mind" is gone. The interface is merely adapted to "a human."